early-attack-detector-py/src/agent.py

```diff
@@ -299,10 +323,10 @@ def detect_malicious_contract(
                 MODEL_THRESHOLD,
                 error=error,
             )
-            if model_score is not None and model_score >= MODEL_INFO_THRESHOLD:
+            if (model_score is not None and model_score >= MODEL_INFO_THRESHOLD) or from_eoa_in_tp_list:
+                from_label_type = "eoa" if from_eoa_in_tp_list else "contract" if is_contract(w3, from_) else "eoa"
```

Comment:
Can be written a bit more concisely
```suggestion
                from_label_type = "eoa" if from_eoa_in_tp_list or not is_contract(w3, from_) else "contract"
```